### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by pabs3**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -2052,6 +2058,7 @@ buiding->building
 buil->build, built,
 build-dependancies->build-dependencies
 build-dependancy->build-dependency
+build-dependencie->build-dependency
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: This can also be `build-dependencies`

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -7018,6 +7051,7 @@ hadling->handling
 haeder->header
 haemorrage->haemorrhage
 haev->have, heave,
+hahve->have
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: could also be `halve, half`

**Comment by pabs3** [comment]
Added both of those.

-- 
bye,
pabs

https://bonedaddy.net/pabs3/

**Comment by luzpaz** [comment]
branch has conflict, needs rebase

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: :+1:

**Comment by pabs3** [comment]
Fixed the conflict.

-- 
bye,
pabs

https://bonedaddy.net/pabs3/

**Comment by larsoner** [comment]
Manually committed to `master` after fixing a seemingly errant dup of `relaesed` that was causing CI failure

**Comment by pabs3** [comment]
Woops, I didn't see that one. Thanks for fixing.

-- 
bye,
pabs

https://bonedaddy.net/pabs3/

**Pull request closed without merging**